http_wrapper.go

```diff
@@ -32,7 +32,7 @@ func run() error {
   mux := runtime.NewServeMux()
   runtime.SetHTTPBodyMarshaler(mux)
 
-  opts := []grpc.DialOption{grpc.WithInsecure()}
+  opts := []grpc.DialOption{grpc.WithInsecure(), grpc.WithMaxMsgSize(10,485,760)}
```

Comment:
It looks like [this](https://pkg.go.dev/google.golang.org/grpc#WithMaxMsgSize) function has been deprecated.

```
Deprecated: use WithDefaultCallOptions(MaxCallRecvMsgSize(s)) instead. Will be supported throughout 1.x.
```

Also, do we need the commas for the int?